Jackson Databind Vulnerability patch

indyaah opened the pull request:
- The actual requirement was to upgrade to `>=2.8.11.1` not `2.8.9` (The first emails were incorrect)

maxhutch commented on pom.xml:
```diff
@@ -83,7 +83,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <jackson.version>[2.8.9]</jackson.version>
+        <jackson.version>[2.8.11.1,)</jackson.version>
```
No upper bound?  `3.0.0` seems easily motivated, unless jackson specifically doesn't follow semver

indyaah replied:
Given that the patched version is `2.8.11.1` they don't see to, but I will nonetheless put an upper bound. It definitely saves us from unintended side-effects.

indyaah commented:
Closing and reopening PR so that Travis picks it up.

indyaah committed:
CITR-2896 :Upgrade 'jackson-databind' to 2.8.11.1
- The actual requirement was to upgrade to 2.8.11.1 not 2.8.9 (The first emails were incorrect)

indyaah committed:
CITR-2896 :Upgrade 'jackson-databind' to 2.8.11.1
- Version bump

EricLundberg commented:
This was approved by Hutch, so I'm executing the merge

EricLundberg reviewed (approved):
This meets the needs of the ticket
